napari/utils/migrations.py

```diff
@@ -28,7 +29,9 @@ def rename_argument(
     if not since_version:
         since_version = "unknown"
         warnings.warn(
-            "The since_version argument was added in napari 0.4.18 and will be mandatory since 0.6.0 release.",
+            trans._(
+                "The since_version argument was added in napari 0.4.18 and will be mandatory since 0.6.0 release."
```

Comment:
```suggestion
                "The since_version argument was added in napari 0.4.18 and will be mandatory since 0.6.0 release.",
                deferred=True
```